### DISCUSSION TITLE
Datagrid Filter

### DISCUSSION THREAD

**Discussion by bastigerner** [Megabit/Blazorise#3202] [Q&A] [answered]
Hey,

quick question: Is it possible to set filterable true in the datagrid and still hide the filter row in the datagrid.
I need each filtercontext from each column fo my filter to work.

Is there any way to get this done?

Kind Regards
Sebastian

**Comment by David-Moreira** [comment, marked as answer]
If you are setting the DataGrid filterable to true, does that not mean that you want to use the Datagrid's filter functionality?
Well.. 
- you could set the filter row display to none. 
- use the customfilter?
https://bootstrapdemo.blazorise.com/tests/datagrid/filter/custom

Not sure if that's what you are looking for.

**Reply by bastigerner** [reply]
> If you are setting the DataGrid filterable to true, does that not mean that you want to use the Datagrid's filter functionality? Well..
> 
> * you could set the filter row display to none.
> * use the customfilter?
>   https://bootstrapdemo.blazorise.com/tests/datagrid/filter/custom
> 
> Not sure if that's what you are looking for.

Thanks the first one works for me. Because I wanted to keep the filtering like I had it before and use the mechanism from the grid.
I hope you understand what and how I mean.
